wp-content/themes/aerospace/assets/_scss/pages/_archives.scss

```diff
@@ -5,204 +5,335 @@
-	        width: calc(100% - #{$archive-content-padding-md});
-			margin: 0 auto;
-		}
+    &-divider {
+      color: $blue-gray;
+      margin: 0 0.5rem;
+    }
+  }
 
-		.archives-meta-left,
-		.archives-meta-right {
-			@include breakpoint("medium") {
-				padding-left: 0;
-				padding-right: 0;
-			}
-		}
-	}
+  .archive-pages-bottom {
+    position: relative;
+    background-color: $white;
+    padding: $archive-content-padding;
 
-    .archives-meta-left {
-        margin-bottom: 5px;
+    @include breakpoint('medium') {
+      padding: $archive-content-padding-md;
+    }
 
-        @include breakpoint("small") {
-            margin-bottom: 0;
-        }
+    &::before {
+      content: '';
+      display: block;
+      position: absolute;
+      top: calc((2 * #{$archive-content-padding}) + 4px);
+      width: calc(100% - (#{$archive-content-padding} * 2));
+      height: 2px;
+      background-color: $blue-gray;
+
+      @include breakpoint('small') {
+        top: calc(1.3rem - 1px);
+      }
+
+      @include breakpoint('medium') {
+        top: calc(#{$archive-content-padding-md} - 1px);
+        width: calc(100% - (#{$archive-content-padding-md} * 2));
+      }
     }
+  }
 
-	.archives-meta-right {
-		@include breakpoint("small") {
-			text-align: right;
-		}
-	}
+  .hentry {
+    background-color: $white;
+    padding: calc(#{$archive-content-padding} + 0.5rem) $archive-content-padding
+      calc(#{$archive-content-padding} - 0.5rem) $archive-content-padding;
 
-	.sort-filter {
-		@include fontSize(16px);
-		line-height: 2.0;
+    @include breakpoint('medium') {
+      padding: $archive-content-padding-md;
+      padding-bottom: 0;
+    }
+  }
 
-		a {
-			color: $black;
-			box-shadow: none;
+  /*-----  Page 2+ Styles or Filtered Results  -------*/
+  &.paged,
+  &.archive-filtered-results {
+    .page-header {
+      margin-top: 0;
+      margin-bottom: 0;
+    }
 
-            &.active,
-			&:hover {
-				color: $orange;
-			}
-		}
+    .page-title {
+      @include fontSize(28px);
+      line-height: 1.71;
+      text-align: left;
+      margin: 0 auto;
+      padding-bottom: 0;
+
+      @include breakpoint('medium') {
+        padding-left: 1rem;
+        padding-right: 1rem;
+        width: calc(100% - 2rem);
+      }
+    }
 
-		&-divider {
-			color: $blue-gray;
-			margin: 0 0.5rem;
-		}
+    .archive-description {
+      display: none;
+    }
+
+    .archive-pages-top {
+      @include breakpoint('medium') {
+        width: calc(100% - #{$archive-content-padding-md});
+        margin: 0 auto;
+      }
+    }
+  }
+}
+
+.archive__featured-posts {
+	background-color: $dark-blue;
+  margin: -8px;
+  margin-bottom: 50px;
+
+	@include breakpoint('medium') {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+	@include breakpoint('large') {
+		margin: 0;
 	}
 
-	.archive-pages-bottom {
-		position: relative;
-		background-color: $white;
-		padding: $archive-content-padding;
+	.col-xs-12 {
```

Comment:
Instead of adjusting the padding here,  I would do something like the below. This changes will get rid of the padding on the larger screens, but keep the default padding on smaller ones, which we want so the posts line up vertically with the non-featured posts below.

```
.archive__featured-posts {
  // styles here

  .entry-main {
    @include breakpoint('medium') {
       padding-right: 0;
       padding-left: 0;
     }
  }
}
```